app/src/main/java/com/example/android/codelabs/paging/api/GithubService.kt

```diff
@@ -53,7 +53,7 @@ suspend fun searchRepos(
         Log.d(TAG, "got a response $response")
         if (response.isSuccessful) {
             val repos = response.body()?.items ?: emptyList()
-            return RepoSearchResult.Success(repos)
+            return RepoSearchResult.Success(repos, response.body()?.total ?: 0)
```

Comment:
What does it look like if you just return the response object directly from the GithubService?

I think removing the RepoSearchResult wrapper may make it easier for folks to see retrofit <-> paging connection all in one place, and copy the code out.

Reply:
If this is only used in PagingSource then, it should be ok. I just don't want to Retrofit `Response` to leak in Repository.